templates/store/search.html

```diff
@@ -0,0 +1,45 @@
+{% extends "_layout.html" %}
+
+{% block title %}Search results for {{ context.query }}{% endblock %}
+
+{% block content %}
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-12 search-results__overview">
+      Your search for <strong>&lsquo;{{ context.query }}&rsquo;</strong> returned
+      {{ context.results_count }} result{{ context.results_count|pluralize }}.
+    </div>
+  </div>
+</div>
+<div class="row">
+  <h4>
+    Recommended
+    <span class="count">
+      ({{ context.results.recommended|length }})
+    </span>
+  </h4>
+  <table class="search-results-table">
+    <thead>
```

Comment:
Why have a thead if it's not doing anything?

Reply:
Hmm... I'll take a look, I just copied over from blues browser.